docs/src/main/sphinx/client.md

```diff
@@ -16,5 +17,6 @@ client/cli
 client/jdbc
 ```
 
-In addition, the community provides [numerous other clients](https://trino.io/resources.html) for platforms such as Python, and these
-can in turn be used to connect applications using these platforms.
+In addition, the community provides [numerous other client libraries, drivers,
```

Comment:
We should separate out the clients maintained directly by the Trino community.

Reply:
Yes .. thats a separate task for the website though .. ping me when you have time to chat about that. I am not sure how to best do that but have a few ideas.